test/server/cards/01_SOR/events/KeepFighting.spec.js

```diff
@@ -0,0 +1,48 @@
+describe('Keep Fighting', function () {
+    integration(function () {
+        describe('Keep Fighting\'s ability', function () {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['keep-fighting'],
+                        groundArena: ['pyke-sentinel', 'wampa'],
+                        leader: { card: 'sabine-wren#galvanized-revolutionary', deployed: true, damage: 4 }
+                    },
+                    player2: {
+                        groundArena: ['atst'],
+                        spaceArena: ['imperial-interceptor']
+                    }
+                });
+            });
+
+            it('should ready a unit', function () {
+                // attack with pyke sentinel (2/3)
+                this.player1.clickCard(this.pykeSentinel);
+                this.player1.clickCard(this.p2Base);
+                expect(this.pykeSentinel.exhausted).toBeTrue();
```

Comment:
Don't really need to have these expectations since they're not related to the card being tested (they're more testing that the attack stuff is working), let's remove them to keep the test as focused as possible